### ISSUE TITLE
Extra folders included

### ISSUE THREAD

**Issue by AnotherCommander** [HiranChaudhuri/oolite-windows-build-env#1]
I just realized that there are some folders present which are not necessarily needed for the dev environment to function. They just take up space. You may want to remove them, as this could maybe help with the out of memory exception you are getting when the github workflow tries to run.

The extra folders are Mingw32_100300, Mingw64_100300, mingw32_710 and mingw64_710. These are newer gcc versions, which are (currently) not utilized during the build of Oolite.

**Comment by HiranChaudhuri** [comment]
I removed the mentioned directories in https://github.com/HiranChaudhuri/oolite-windows-build-env/commit/76edbe04b799de47a363c8735baf361249fd4fbe

**Comment by HiranChaudhuri** [comment]
Good catch. Now that the files are somewhat reduced, creating the zip actually succeeded:
https://github.com/HiranChaudhuri/oolite-windows-build-env/releases/tag/latest
